### PR TITLE
[#40267] Opening viewpoints on mobile

### PR DESCRIPTION
**Pull request by Kharonus**
- https://community.openproject.org/work_packages/40267
- removed mobile alternative for details routes for bcf module
- added style for details page on mobile resolutions

### CODE REVIEW

File: frontend/src/global_styles/layout/work_packages/_details_view.sass [hunk 1]
```diff
@@ -48,6 +48,11 @@ body.router--work-packages-partitioned-split-view-new
   // Min-width is actually 530px but the border already needs 2px
   min-width: 528px
 
+  // Special handling for BCF module, which actually displays the details even on a mobile screen
+  @media only screen and (max-width: 679px)
+    op-ifc-viewer-page &
```
Comment (HDinger) [inline frontend/src/global_styles/layout/work_packages/_details_view.sass]: We usually prefer classes over tag selectors. Further I would suggest to move this code into its own (global) sass file, similar to how it is done for the [team_planner](https://github.com/opf/openproject/blob/dev/frontend/src/global_styles/content/modules/_team_planner.sass). If you set a `router--bim`(?) class in the routes, you can access this in that file and define all relevant global bim styles. This has the advantage that it is centralised in one place and not distributed all over the code.

**Review by HDinger — COMMENTED** [review]
Works fine 👍 I only have one minor comment regarding code placement.

File: frontend/src/global_styles/layout/work_packages/_details_view.sass (hunk 1 above)
Comment (Kharonus) [reply frontend/src/global_styles/layout/work_packages/_details_view.sass]: Fixed that, introduced body class `router--bim` on the base route in the bcf module.

**Comment by Kharonus** [comment]
Slightly amended the bim-navigation spec to cover the issue, that a click on a wp-card's ui-state-link behaves correctly.

**Pull request merged**